src/runtime_src/core/common/query_requests.h

```diff
@@ -32,6 +32,14 @@ namespace xrt_core {
 
 namespace query {
 
+enum bdf
```

Comment:
Being that these values are used to get in an "index" value,  it is best to:
1. Describe that this enumeration represents.
2. Initialize each value instead of relying on the default behavior.
3. Add a 1 line comment at the initializing of the enumeration value indicating purpose and an example value

Again, the purpose is readability and helping the next developer support this code base.